glotaran/analysis/problem.py

```diff
@@ -244,7 +243,9 @@ def full_penalty(self) -> np.ndarray:
                     additional_penalty[label] for label in additional_penalty.keys()
```

Comment:
Since `dict` has the `values` method line 242-245, can be rewritten as
```python
residuals = [np.concatenate(residual) for residual in residuals.values()]
additional_penalty = list(additional_penalty.values())
```

Reply:
true